src/libcsg/csgapplication.cc

```diff
@@ -34,31 +34,31 @@ void CsgApplication::Initialize() {
   TopologyReader::RegisterPlugins();
 
   if (NeedsTopology()) {
-    AddProgramOptions()("top", boost::program_options::value<string>(),
+    AddProgramOptions()("top", boost::program_options::value<std::string>(),
```

Comment:
Again why not use 'using namespace std;' here in the source files?

Reply:
as I said, I am not a big fan of using namespace and it is more explicit this way.